lib/stripe/charges.ex

```diff
@@ -292,14 +292,28 @@ defmodule Stripe.Charges do
 
   """
   def refund(id) do
-    refund id, Stripe.config_or_env_key
+    refund id, Stripe.config_or_env_key, %{}
   end
 
   @doc """
+  Refunds a charge and reverses transfer to connected accounts.
+
+  Return a `{:ok, charge}` tuple.
+
+  ## Examples
+
+      {:ok, charge} = Stripe.Charges.refund_with_reversal("charge_id")
+  """
+  def refund_with_reversal(id) do
+    refund id, Stripe.config_or_env_key, %{reverse_transfer: true, refund_application_fee: true}
+  end
```

Comment:
I'm unsure about the convenience method. 

It absolutely is convenient, but considering we are actively working on 2.0 and sort of holding `master` in a feature freeze state, adding such methods just means more to cover later.  We would prefer to limit changes in master to primarily bug fixes.

Anyone who needs a specialised method such as this one can easily define it themselves, as you did in your application. 

That being said, if you feel the method is common enough to be introduced into the library, would you mind elaborating? 

Could use thoughts by other people to. @DavidAntaramian @dmarkow @JoshSmith